spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../lib/takeaway.rb'
+require_relative '../../lib/menu.rb'
+require 'twilio-ruby'
+
+describe "FEATURE SPEC" do
+  let(:takeaway) { Takeaway.new(Menu.new) }
+  let(:choice) { {beef:2, rolls:3} }
+  let(:default_menu) { {ribs: 3,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.